congressional-support.yaml

```diff
@@ -0,0 +1,119 @@
+- type: house
+  name: House Documents Repository
+  url: http://docs.house.gov/
+
+- type: house
+  name: Office of Compliance
+  url: http://www.compliance.gov/
+
+- type: house
+  name: Clerk of the House
+  url: http://clerk.house.gov/
+
+- type: house
+  name: Chief Administrative Officer
+  url: http://cao.house.gov/
+
+- type: house
+  name: Sergeant at Arms
+  url:
+
+- type: house
+  name: Chaplain
+  url: http://chaplain.house.gov/
+
+- type: house
+  name: Inspector General
+  url: http://www.house.gov/content/learn/officers_and_organizations/inspector_general.php/
+
+- type: house
+  name: Historian
+  url: http://history.house.gov/
+
+- type: house
+  name: Law Revision Counsel
+  url: http://uscode.house.gov/
+
+- type: house
+  name: Legislative Counsel
+  url: http://legcounsel.house.gov/
+
+- type: house
+  name: Office of Congressional Ethics
+  url: http://oce.house.gov/
+
+- type: house
+  name: Office of the General Counsel
+  url: http://www.ogc.house.gov/
+
+- type: house
+  name: Office of Interparliamentary Affairs
+  url:
+
+- type: house
+  name: Parliamentarian
+  url:
+
+- type: house
+  name: Bipartisan Legal Advisory Group
+  url:
+
+- type: support agency
```

Comment:
For a `type` field, my personal preference is `support_agency` with an underscore, to signal it's a slug and not a free text field. This is aesthetic and not a blocker.